docs/compressed-serialization.md

```diff
@@ -0,0 +1,196 @@
+# Compressed Serialization
+
+With the Chia hard fork at height 5'496'000, CLVM can be serialized in a more
+space efficient form, referring back to previous sub-trees instead of
```

Comment:
- space efficient form, referring back to previous sub-trees instead of
+ space efficient form by referring back to previous sub-trees instead of